fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -193,6 +210,12 @@ public void writeFDBLuceneFileReference(@Nonnull final String name, @Nonnull fin
      * @param value the data to be stored
      */
     public void writeData(long id, int block, @Nonnull byte[] value) {
+        FDBStoreTimer timer = context.getTimer();
+        if (timer != null) {
+            timer.increment(FDBStoreTimer.Counts.LUCENE_DATA_WRITE);
+            //This may not be correct transactionally
+            timer.increment(FDBStoreTimer.Counts.LUCENE_DATA_SIZE, value.length);
```

Comment:
There is no need for two metrics here, you can just send `LUCENE_DATA_SIZE`.  In Hubble, plotting the `count/sec` of the `LUCENE_DATA_SIZE` will implicitly be the number of writes.  You may, however, want to change the metric to `LUCENE_WRITE_SIZE` to clarify the meaning of the metric.